utils/coverpage.tex

```diff
@@ -0,0 +1,118 @@
+\thispagestyle{empty}
```

Comment:
Why are there `content/coverpage.tex` _and_ `utils/coverpage.tex` files?

Reply:
Obviously I uploaded too many files. I can delete them and add them to another pull request or you can delete them.

Reply:
The coverage and utils should be a different pull. I also did some trivial mods to programming model.

Reply:
coverpage